### PR TITLE
[MRG] Adding name to zenodo and adding DOI badge to README

### PR DESCRIPTION
**Pull request by adam2392**
PR Description
--------------

See: #460 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"
- [ ] Commit history does not contain any merge commits

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,6 +31,10 @@
   :target: https://joss.theoj.org/papers/5b9024503f7bea324d5e738a12b0a108
   :alt: JOSS publication
 
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3739558.svg
+  :target: https://zenodo.org/badge/DOI/10.5281/zenodo.3739558.svg
+  :alt: DOI
```
Comment (adam2392) [inline README.rst:36]:
```suggestion
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3739558.svg
   :target: https://doi.org/10.5281/zenodo.3739558
   :alt: DOI
```

File: README.rst [hunk 2]
```diff
@@ -31,6 +31,10 @@
   :target: https://joss.theoj.org/papers/5b9024503f7bea324d5e738a12b0a108
   :alt: JOSS publication
 
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3739558.svg
```
Comment (jasmainak) [inline README.rst:34]: is this zenodo batch different from the paper zenodo?

Comment (sappelhoff) [reply README.rst:34]: They are separate:
- JOSS stores the paper and mints a DOI
- JOSS links to the zenodo archive that stores the code at the version when the JOSS paper was published

you can get to Zenodo from JOSS by clicking on "Software Archive" (see screenshot):

image

**Comment by jasmainak** [comment]
@adam2392 I am sorry I didn't realize you were not part of the paper. I think it would be very hard to add you to the paper / citations now :-( We can of course add you to the zenodo but it will confuse people if there are two citeables. I hope that learning git/github and getting feedback on code is a fair incentive, at least for now :-) And we'll acknowledge your contributions everywhere

**Comment by adam2392** [comment]
> @adam2392 I am sorry I didn't realize you were not part of the paper. I think it would be very hard to add you to the paper / citations now :-( We can of course add you to the zenodo but it will confuse people if there are two citeables. I hope that learning git/github and getting feedback on code is a fair incentive, at least for now :-) And we'll acknowledge your contributions everywhere

No problem honestly and yeah of course I understand that. 

Feedback and contributing to the open-science community has been fairly incentivizing so far :). I've been in my PhD too long and seen so many failures of datasets curation/maintenance that I'm glad to see a package like this maturing. P.S. also, basically tell every collaborator to use this now and switch from the bane of matlab.

Fixed the DOI badge.

**Comment by jasmainak** [comment]
Thanks @adam2392 for the PR and the contributions so far!!

**Pull request merged**